tests/lwip/Makefile

```diff
@@ -1,4 +1,5 @@
 APPLICATION = lwip
+include ../Makefile.tests_common
```

Comment:
I don't know about the other tests, but this was left out intentionally to set `BOARD` below properly (fixed in #6372).